### PR TITLE
cli: inspect and refactor wallet-related commands

### PR DESCRIPTION
**Pull request by AnnaShaleva**
This PR unifies wallet-related commands descriptions and arguments.

### CODE REVIEW

File: cli/wallet/wallet.go [hunk 1]
```diff
@@ -164,10 +164,6 @@ func NewCommands() []cli.Command {
 						Name:  "name, n",
 						Usage: "Optional account name",
 					},
-					cli.StringFlag{
-						Name:  "contract",
-						Usage: "Verification script for custom contracts",
-					},
```
Comment (roman-khimov) [inline cli/wallet/wallet.go:170]: This for non-standard non-deployed contracts, it has to be supported.

File: cli/wallet/wallet.go [hunk 2]
```diff
@@ -206,11 +206,15 @@ func NewCommands() []cli.Command {
 			{
 				Name:      "remove",
 				Usage:     "remove an account from the wallet",
-				UsageText: "remove --wallet <path> [--force] <addr>",
+				UsageText: "remove --wallet <path> [--force] --account <addr>",
```
Comment (roman-khimov) [inline cli/wallet/wallet.go:210]: Breaking change, although it's not a frequently used command, so probably can afford it.

File: cli/wallet/wallet.go (hunk 1 above)
Comment (AnnaShaleva) [reply cli/wallet/wallet.go]: Removed this commit. Just a question: why do we need this, if we can only work with standard contracts signatures?

File: cli/wallet/wallet.go (hunk 2 above)
Comment (AnnaShaleva) [reply cli/wallet/wallet.go:214]: It's just for unification. Do we have to be compatible with 2x cli?

Comment (fyrchik) [reply cli/wallet/wallet.go:214]: Also, we use `--address` for such options.

Comment (roman-khimov) [reply cli/wallet/wallet.go:214]: 2.x is irrelevant, it's just that we already have some scripts written for current syntax, so we can't arbitrarily change CLI. At the same time, this command shouldn't be used by that scripts, so it's safe to change.

Comment (AnnaShaleva) [reply cli/wallet/wallet.go]: Fixed.

File: cli/wallet/wallet.go (hunk 1 above)
Comment (roman-khimov) [reply cli/wallet/wallet.go]: Verification scripts can contain any NeoVM code and we need to support it. Maybe we'll extend this support in future, but for now at least we need to be able to import these scripts into wallet.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**